enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -27,32 +29,34 @@ def now():
     return UTC.localize(datetime.utcnow())
 
 
-class TimeStampedModelWithUuid(TimeStampedModel):
-    class Meta:
-        abstract = True
-
-    uuid = models.UUIDField(
-        primary_key=True,
-        default=uuid4,
-        editable=False,
-        unique=True,
-    )
-
-
-class Subsidy(TimeStampedModelWithUuid):
+class Subsidy(TimeStampedModel):
     """
     Some defintions:
     ``ledger``: A running “list” of transactions that record the movement of value in and out of a subsidy.
     ``stored value``: Value, in the form of a subscription license (denominated in “seats”)
         or learner credit (denominated in either USD or "seats"), stored in a ledger,
         which may be applied toward the cost of some content.
     ``redemption``: The act of redeeming stored value for content.
-
-    
     """
     class Meta:
+        """
+        Metaclass for Subsidy.
+        """
         abstract = True
 
+    uuid = models.UUIDField(
+        primary_key=True,
+        default=uuid4,
+        editable=False,
+        unique=True,
+    )
+    # `title` is required for downstream revenue recognition.  The intention is to serve as a replacement to "offer
```

Comment:
what is an example of a value here?

Reply:
Here's a random sampling of 5 offer_names from the snowflake table `prod.core.dim_offer`:

```
OFFER_NAME
Coupon [8160]-Percentage-20.00 [1100]
Coupon [12091]-Absolute-450.00 [291]
Coupon [63226]-Percentage-100 [155] ENT Offer
Coupon [85123]-Percentage-100 [40]
Coupon [75299]-Percentage-100 [12] ENT Offer
```

Reply:
Perusing the edx-ecommerce source code, the actual definition and setting of this field seems to live in the depths of the Oscar source code.  On the surface it looks like a rendering of internal metadata about the offer, which would make this type of field better suited as a property than a persisted database column. :shrug: